platform/linux-generic/include/odp_packet_internal.h

```diff
@@ -153,13 +152,10 @@ typedef struct odp_packet_hdr_t {
 	uint8_t data[];
 
 } odp_packet_hdr_t;
-#pragma GCC diagnostic pop
 
 /* Packet header size is critical for performance. Ensure that it does not accidentally
- * grow over 256 bytes when cache line size is 64 bytes (or less). With larger cache line sizes,
- * the struct size is larger due to the odp_buffer_hdr_t alignment requirement. */
-ODP_STATIC_ASSERT(sizeof(odp_packet_hdr_t) <= 256 || ODP_CACHE_LINE_SIZE > 64,
-		  "PACKET_HDR_SIZE_ERROR");
+ * grow over 256 bytes when cache line size is 64 bytes (or less). */
+ODP_STATIC_ASSERT(sizeof(odp_packet_hdr_t) <= 256, "PACKET_HDR_SIZE_ERROR");
```

Comment:
"...when cache line size is 64 bytes (or less)." comment can be removed also

Reply:
Fixed in V2.